README.rst

```diff
@@ -1024,12 +1029,9 @@ See also: `CONTRIBUTING.md <https://github.com/falconry/falcon/blob/master/CONTR
 Legal
 -----
 
-Copyright 2013-2020 by Individual and corporate contributors as
+Copyright 2013-2021 by Individual and corporate contributors as
```

Comment:
2022?

Reply:
It's not 2022 yet :slightly_smiling_face: So let's not put the cart before the horse.